generatebundlefile/bundle.go

```diff
@@ -95,28 +98,61 @@ func AddMetadata(s api.PackageBundleSpec) BundleGenerate {
 	}
 }
 
+func CheckSignature(bundle *api.PackageBundle) (bool, error) {
```

Comment:
I tried doing the receiver method, but it seemed to complicate thing since I'd need to interface's for each of the API types from the controller API, which might be tougher when there is changes later.

I will totally fix/change it if you need though.